rust/sysdb/src/sqlite.rs

```diff
@@ -1261,6 +1305,15 @@ mod tests {
             metadata.get("key1").unwrap(),
             &MetadataValue::Str("value1".to_string())
         );
+
+        // Access HNSW configuration through pattern matching
+        match &collection.config.vector_index {
```

Comment:
check out the `matches!()` macro instead, it's slightly more idiomatic for this kind of usage